handout/redux/review_of_reducers_and_pure_functions.md

```diff
@@ -1,8 +1,8 @@
 # Review of Reducers and Pure Functions
```

Comment:
Not a reason to block this PR, but this section seems to call for at least of tiny bit of extension. (Maybe we should just file this as an issue?)